### PR TITLE
NEW FEATURE: Add RFC4183 support to REV()

### PR DESCRIPTION
**Pull request by tlimoncelli**
CC @misilot
Fixes https://github.com/StackExchange/dnscontrol/issues/2844
Fixes https://github.com/StackExchange/dnscontrol/issues/2845

### CODE REVIEW

**Comment by tlimoncelli** [comment]
Hey, @misilot !  Could you confirm this addresses the feature request you made?

Thanks!

File: documentation/functions/global/REV.md [hunk 1]
```diff
@@ -10,47 +10,68 @@ ts_return: string
...
-Note that the lower bits (the ones outside the netmask) must be zeros. They are not
-zeroed out automatically. Thus, `REV("1.2.3.4/24")` is an error.  This is done
-to catch typos.
+# RFC compliance
+
+`REV()` implements both RFC 2317 and the newer RFC 4183. The `REVCOMPAT()`
+function selects which mode is used. If `REVCOMPAT()` is not called, a default
+is selected for you.  The default will change to RFC 4183 in DNSControl v5.0.
+
+See [REVCOMPAT()](functions/global/REVCOMPAT.md) for details.
```
Comment (cafferata) [inline documentation/functions/global/REV.md:37]:
```suggestion
See [REVCOMPAT()](REVCOMPAT.md) for details.
```

**Comment by misilot** [comment]
@tlimoncelli I think this looks great reading through the Files Changed! I'm excited for this. Is there a great way to test this out?

**Comment by tlimoncelli** [comment]
[misilot](https://github.com/misilot) friendly ping?  (I'll be doing a release this weekend and would like to include this PR)

**Comment by tlimoncelli** [comment]
Awesome!  Glad it's all working!

**Pull request merged**